games-simulation/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="Simulator Construction Gear"
+DESCRIPTION="Set of open-source libraries designed \
+as building blocks for quickly assembling 3D simulations, \
+games, and visualization applications."
+HOMEPAGE="https://www.flightgear.org"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
+	lib:libsimgear$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenscenegraph$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libplib$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	simgear$secondaryArchSuffix = $portVersion
+	devel:libsimgear$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	simgear$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libopenscenegraph$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libplib$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc
+	cmd:ld
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. \
+		-DSIMGEAR_SHARED:BOOL=ON \
+		-DENABLE_TESTS:BOOL=OFF

```

Comment:
You need to pass extra parameters to cmake to let it know that the libs should go in `$libDir`, otherwise the pc file will be incorrect if I'm not mistaken.
We have lots of recipes using cmake. I'll let you dig :)